Services/Auth0toAI/Auth0toAI/Service/Auth0Service.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+
+namespace Auth0toAI.Service
+{
+    public class Auth0Service
+    {
+        private IConfiguration _config;
+        public TelemetryClient telemetryClient;
+        public Auth0Service(IConfiguration config)
+        {
+            _config = config;
+
+            var telemetryConfiguration = TelemetryConfiguration.CreateDefault();
```

Comment:
default Configuration may include DiagnosticModule and DependencyModule which would add other info to the target ApplicationInsights instance. Please check that's not the case and Default do not register any Initializer and any Module